legislators-current.yaml

```diff
@@ -34606,6 +34608,7 @@
     district: 2
     party: Republican
     phone: 202-225-1621
+    fax: 202-225-2563 
```

Comment:
This (for some reason) broke the build: https://travis-ci.org/unitedstates/congress-legislators/builds/201021210

```
The command "python test/are_files_linted.py" exited with 1.
```

```
$ python test/are_files_linted.py
legislators-current.yaml needs to be linted:
--- in repository
+++ after linting
@@ -34852,7 +34852,7 @@
     district: 2
     party: Republican
     phone: 202-225-1621
-    fax: 202-225-2563 
+    fax: 202-225-2563
     url: http://wagner.house.gov
     rss_url: http://wagner.house.gov/rss.xml
     address: 435 Cannon HOB; Washington DC 20515-2502
```

I'm not sure why - could there be a tab instead of spaces, or something?